lisp/ess-custom.el

```diff
@@ -2108,27 +2113,26 @@ for help files.  The default value is nil for other systems."
 
 ;;;;; names for communication using MS-Windows 9x/NT ddeclient mechanism
 
-(defcustom inferior-ess-ddeclient nil
+(defcustom inferior-ess-ddeclient "Initial"
```

Comment:
let's put the cleanup part in another branch and merge developer2 into master, then we can all work on it from there. Any objection @vspinu ?